watson_embed_model_packager/setup_build_config.py

```diff
@@ -652,6 +655,12 @@ def main(parent_parser: Optional[argparse.ArgumentParser] = None):
         default=str_list_from_env("LIBRARY_VERSIONS"),
         help="lib:version pair(s) for watson libraries being released",
     )
+    parser.add_argument(
+        "--image-tag",
+        "-it",
```

Comment:
hah, this evokes the `-i` + `-t` interactive tty docker run settings in my brain

Reply:
I know!!! but the `-t` is taken! Any other suggestion?